hack/verify-godeps.sh

```diff
@@ -39,9 +39,15 @@ preload-dep() {
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-branch="${1:-master}"
+readonly branch="origin/${1:-${KUBE_VERIFY_GIT_BRANCH:-master}}"
+echo "Checking for Godeps changes against ${branch}"
+# make sure the branch is valid, otherwise the next check will pass erroneously.
+if ! git describe "${branch}" >/dev/null; then
+  exit 1
+fi
 # notice this uses ... to find the first shared ancestor
-if ! git diff origin/"${branch}"...HEAD | grep 'Godeps/' > /dev/null; then
+if ! git diff --name-only "${branch}...HEAD" | grep 'Godeps/' > /dev/null; then
```

Comment:
I'm going to to have to take a look at how this works (or can break) during the git post commit hook...  I'm worried about my local master branch being way way behind the upstream master branch.  (but obviously hard coding 'origin' didn't work either)